migrations/202010171600125-update-handler-ids-add-timestamp.js

```diff
@@ -4,11 +4,11 @@ module.exports = {
             'handler_ids',
             'timestamp',
             {
-                type: Sequelize.INTEGER,
```

Comment:
Are these types compatible with both mysql and postgresql implementations?

Reply:
Yes, i've tested both of them